### PR TITLE
expression: add benchmarks about expression executor

### PR DESCRIPTION
**Pull request by zz-jason**

### CODE REVIEW

File: expression/chunk_executor_test.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package expression
```
Comment (shenli) [inline expression/chunk_executor_test.go]: Please add a few comment in this file.

File: expression/chunk_executor_test.go [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+	h.inputTypes = append(h.inputTypes, &types.FieldType{
+		Tp:      mysql.TypeNewDecimal,
+		Flen:    11,
+		Decimal: 0,
+		Flag:    mysql.BinaryFlag,
+		Charset: charset.CharsetBin,
+		Collate: charset.CollationBin,
+	})
+
+	for i := 0; i < 20; i++ {
```
Comment (coocood) [inline expression/chunk_executor_test.go]: Why 20 string types?

File: expression/chunk_executor_test.go [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		h.outputChunk.Reset()
+		if err := VectorizedExecute(h.ctx, h.exprs, inputIter, h.outputChunk); err != nil {
+			panic("errors happend during \"VectorizedExecute\"")
+		}
+	}
+}
+
+func BenchmarkUnVectorizedExecute(b *testing.B) {
```
Comment (winoros) [inline expression/chunk_executor_test.go]: It seems that no where use the `UnVectorizedExecute`? Can we just remove it?

File: expression/chunk_executor_test.go (hunk 2 above)
Comment (zz-jason) [reply expression/chunk_executor_test.go]: to show the power of cache performance.

**Comment by zz-jason** [comment]
@coocood @winoros @shenli PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**